dev/com.ibm.ws.threading/src/com/ibm/ws/threading/internal/ThroughputDistribution.java

```diff
@@ -47,22 +47,74 @@
      */
     double ewmaVariance = Double.NEGATIVE_INFINITY;
 
+    /**
+     * The standard deviation of the observed data.
+     * <p>
+     * The value {@code Double.NEGATIVE_INFINITY} indicates no observations
+     * have been made.
+     */
+    double ewmaStddev = Double.NEGATIVE_INFINITY;
+
+    /**
+     * The last controller cycle when this distribution was updated. The controller
+     * will discard old data.
+     * This is an int - at 1500ms controller cycles, that allows for 100+ years of
+     * continuous controller operation (no restarts) before it would recycle.
+     */
+    int lastUpdate = 0;
+
+    synchronized void setLastUpdate(int update) {
+        lastUpdate = update;
+    }
+
+    synchronized int getLastUpdate() {
```

Comment:
method does not appear to be used

Reply:
Thanks for the feedback Nathan. 
Methods in this class are used by ThreadPoolController, which is in the same package. getLastUpdate() is used by the changes I'm making now in ThreadPoolController, which should be committed in the next few days.